From b247f63441f75c47cebd6edd00ecc0c5f94728b6 Mon Sep 17 00:00:00 2001
From: Marius Vlad <marius.vlad@collabora.com>
Date: Tue, 2 Jan 2024 15:00:13 +0200
Subject: [PATCH 3/3] activation_area: Remove the implicit width/height swap

This does more harm than being useful so remove it.

Bug-AGL: SPEC-5038
Signed-off-by: Marius Vlad <marius.vlad@collabora.com>
---
 shell/wayland/display.cc | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/shell/wayland/display.cc b/shell/wayland/display.cc
index 0b97954..9452dbf 100644
--- a/shell/wayland/display.cc
+++ b/shell/wayland/display.cc
@@ -863,12 +863,6 @@ void Display::AglShellDoSetupActivationArea(uint32_t x,
   if (!m_agl.shell)
     return;
 
-  if (m_all_outputs[index]->transform == WL_OUTPUT_TRANSFORM_90) {
-    uint32_t tmp_width = width;
-    width = height;
-    height = tmp_width;
-  }
-
   SPDLOG_DEBUG("Using custom rectangle [{}x{}+{}x{}] for activation", width,
                height, x, y);
 
-- 
2.35.1

